xo-game/src/Components/Start/Start.jsx

```diff
@@ -21,8 +21,10 @@ export default function Start() {
   const handleAddName = (name) => {
     if (activePlayer === "X") {
       setPlayerX(name);
+      handlePlayerClick("O")
     } else if (activePlayer === "O") {
       setPlayerO(name);
+      handlePlayerClick("X")
```

Comment:
remove this just move from the fitrst player to the second
don't return back to the first one, but the userr can return back if he wanted to change the name